src/main/java/com/volmit/iris/Iris.java

```diff
@@ -443,40 +443,38 @@ public void imsg(CommandSender s, String msg) {
 
     @Override
     public ChunkGenerator getDefaultWorldGenerator(String worldName, String id) {
-        String dimension = IrisSettings.get().getGenerator().getDefaultWorldType();
 
-        if (id != null && !id.isEmpty()) {
-            dimension = id;
-            Iris.info("Generator ID: " + id + " requested by bukkit/plugin. Assuming IrisDimension: " + id);
+        IrisDimension dim;
+        if (id == null || id.isEmpty()){
+            dim = IrisData.loadAnyDimension(IrisSettings.get().getGenerator().getDefaultWorldType());
+        } else {
+            dim = IrisData.loadAnyDimension(id);
         }
+        Iris.info("Generator ID: " + id + " requested by bukkit/plugin");
```

Comment:
This should be debug, its useless information for those not in debug or verbose... but we are slowly dropping support for verbose so it should be debug.